# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1491570794 14400
#      Fri Apr 07 09:13:14 2017 -0400
# Node ID ffbeda5a6502cd3ec72b5dd3afcaf19509708208
# Parent  7fbf0d4e48c99d7837d20da39b2c7853d4f5f919
* audiodevinfo (Faudiodevinfo): Improve doc string.

diff --git a/libinterp/dldfcn/audiodevinfo.cc b/libinterp/dldfcn/audiodevinfo.cc
--- a/libinterp/dldfcn/audiodevinfo.cc
+++ b/libinterp/dldfcn/audiodevinfo.cc
@@ -84,17 +84,19 @@ DEFUN_DLD (audiodevinfo, args, ,
 Return a structure describing the available audio input and output devices.
 
 The @var{devinfo} structure has two fields @qcode{"input"} and
 @qcode{"output"}.  The value of each field is a structure array with
 fields @qcode{"Name"}, @nospell{"DriverVersion"} and @qcode{"ID"}
 describing an audio device.
 
 If the optional argument @var{io} is 1, return information about input
-devices only.  If it is 0, return information about output devices only.
+devices only.  If it is 0, return information about output devices
+only.  If @var{io} is the only argument supplied, return the number of
+intput or output devices available.
 
 If the optional argument @var{id} is provided, return information about
 the corresponding device.
 
 If the optional argument @var{name} is provided, return the id of the
 named device.
 
 Given a sampling rate, bits per sample, and number of channels for an input
